examples/bri-3/src/bri/communication/agents/messaging.agent.ts

```diff
@@ -53,16 +53,33 @@ export class MessagingAgent implements OnApplicationBootstrap {
       return false;
     }
 
-    return await this.commandBus.execute(
-      new ProcessInboundBpiMessageCommand(
-        newBpiMessageCandidate.id,
-        newBpiMessageCandidate.fromBpiSubjectId,
-        newBpiMessageCandidate.toBpiSubjectId,
-        JSON.stringify(newBpiMessageCandidate.content),
-        newBpiMessageCandidate.signature,
-        newBpiMessageCandidate.type,
-      ),
-    );
+    if (newBpiMessageCandidate.isInfoMessage()) {
```

Comment:
can this condition be checked in command handler since in both cases same command is called and that way we can isolate handling of different types on one place?

Reply:
It is not the same command.

IsInfoMessage -> ProcessInboundBpiMessageCommand
isTransationMessage -> ProcessInboundBpiTransactionCommand